specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/networkVirtualAppliance.json

```diff
@@ -604,6 +604,237 @@
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "networkVirtualApplianceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Network Virtual Appliance."
+          },
+          {
+            "name": "connectionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the NVA connection."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Delete successful.NVA Connection deleted."
+          },
+          "202": {
+            "description": "Accepted, and the operation will continue asynchronously."
+          },
+          "204": {
+            "description": "No NVA connections exist by the name provided."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkVirtualAppliances/{nvaName}/networkVirtualApplianceConnections": {
+      "get": {
+        "operationId": "NetworkVirtualApplianceConnections_List",
+        "x-ms-examples": {
+          "NetworkVirtualApplianceConnectionList": {
+            "$ref": "./examples/NetworkVirtualApplianceConnectionList.json"
+          }
+        },
+        "description": "Lists NetworkVirtualApplianceConnections under the NVA.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "networkVirtualApplianceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Network Virtual Appliance."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Lists all the NVA connections under an NVA.",
+            "schema": {
+              "$ref": "./virtualWan.json#/definitions/NetworkVirtualApplianceConnection"
```

Comment:
>NetworkVirtualApplianceConnection [](http://example.com/codeflow?start=54&length=33)

I think, you meant to put "NetworkVirtualApplianceConnectionList" here, but have put "NetworkVirtualApplianceConnection", right?

Reply:
Yes. Thanks for pointing out. Will fix this.